packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -1078,18 +1085,18 @@ contract Colosseum is Initializable, ISemver {
     }
 
     /**
-     * @notice Returns the challenge corresponding to the given L2 output index.
+     * @notice Returns the challenge segments corresponding to the given L2 output index and challenger.
      *
      * @param _outputIndex Index of the L2 checkpoint output.
      * @param _challenger  Address of the challenger.
      *
-     * @return The challenge data.
+     * @return The challenge segments data.
      */
-    function getChallenge(
+    function getSegments(
```

Comment:
It makes sense to remove this function because the `challenges` mapping is public, but I don't understand why segments getter function is needed.

Reply:
Please refer [this comment](https://github.com/kroma-network/kroma/pull/373#discussion_r1726460992).